tests/unit/AddEditOrgPerson.spec.ts

```diff
@@ -319,8 +319,8 @@ describe('Add/Edit Org/Person component', () => {
 
     const messages = wrapper.findAll('.v-messages__message')
     expect(messages.length).toBe(3)
-    expect(messages.at(0).text()).toBe('Cannot exceed 30 characters')
-    expect(messages.at(1).text()).toBe('Cannot exceed 30 characters')
+    expect(messages.at(0).text()).toBe('Cannot exceed 20 characters')
+    expect(messages.at(1).text()).toBe('Cannot exceed 20 characters')
     expect(messages.at(2).text()).toBe('Cannot exceed 30 characters')
```

Comment:
Please add a set of unit tests to verify this for firms vs others.

Reply:
added unit test specific to firms